policyengine_us/parameters/gov/states/ny/tax/income/deductions/itemized/phase_out/start.yaml

```diff
@@ -1,18 +1,30 @@
 description: Federal AGI over this threshold phases out itemized deductions for NY State income tax.
 SINGLE:
   2021-01-01: 282_400
+  2023-01-01: 313_200
 JOINT:
-  2021-01-01: 310_600
+  2021-01-01: 338_850
+  2023-01-01: 375_850
 SURVIVING_SPOUSE:
-  2021-01-01: 310_600
+  2021-01-01: 338_850
+  2023-01-01: 375_850
 SEPARATE:
   2021-01-01: 169_400
+  2023-01-01: 187_900
 HEAD_OF_HOUSEHOLD:
   2021-01-01: 310_600
+  2023-01-01: 344_500
 metadata:
+  uprating: 
+    parameter: gov.irs.uprating
+    rounding:
+      type: nearest
+      interval: 25
   unit: currency-USD
   period: year
-  label: NY itemized deduction phase out start
+  label: New York itemized deduction phase out start
   reference:
-    - title: NY Form IT-196 Instructions, page 18
-      href: https://www.tax.ny.gov/pdf/current_forms/it/it196i.pdf#page=18
+    - title: 2021 NY Form IT-196 Instructions Line 40
+      href: https://www.tax.ny.gov/pdf/2021/inc/it196i_2021.pdf#page=18
+    - title: 2023 NY Form IT-196 Instructions Line 40
+      href: https://www.tax.ny.gov/pdf/2023/printable-pdfs/inc/it196i-2023.pdf#page=25
```

Comment:
could you add the legal code here?

Reply:
Also could you add the 2022 values as well?

Reply:
I think the state does a very similar thing where they continue to increase the federal reduction amount even if it was removed in the federal system:
see virginia:
policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/applicable_amount.yaml

    # Virginia continued to increase the federal applicable amount with inflation 
    - title: 26 U.S. Code § 68 - Overall limitation on itemized deductions (b)(1)
      href: https://www.law.cornell.edu/uscode/text/26/68#b_1